src/interface.c

```diff
@@ -4,11 +4,9 @@
 
 
 void opart_gaussian_interface
-  (int* n_data, double *data_ptr, double* penalty, double *cost_ptr, int *end_ptr,
-   double* sums, double* sq_sums){
```

Comment:
it would make memory management easier if you keep these `sums` and `sq_sums` arguments (thereby letting R handle the dynamic memory allocation for you)